modules/basics/src/main/java/com/opengamma/strata/basics/market/ReferenceData.java

```diff
@@ -12,26 +12,28 @@
 import com.opengamma.strata.collect.Messages;
 
 /**
- * Provides access to reference data, such as securities and time-series.
+ * Provides access to reference data, such as holiday calendars and securities.
  * <p>
  * Reference data is looked up using implementations of {@link ReferenceDataId}.
- * The result is an instance of {@link TypedReferenceData} that allows multiple pieces
- * of reference data to be associated with the same identifier, based on the data type.
+ * The identifier is parameterized with the type of the reference data to be returned.
  * <p>
  * The standard implementation is {@link ImmutableReferenceData}.
  */
 public interface ReferenceData {
 
   /**
-   * Obtains an instance from a map of typed reference data.
+   * Obtains an instance from a map of reference data.
    * <p>
-   * Each identifier in the map refers to an instance of {@link TypedReferenceData}.
-   * This allows multiple values to be connected to the same identifier, accessed by type.
+   * Each entry in the map is a single piece of reference data, keyed by the matching identifier.
```

Comment:
Perhaps "a _unit_ of reference data" might make the holiday calendar example clearer. I'd think of a _piece_ of data as something finer grained.